src/cf-tables/usage.md

```diff
@@ -7,16 +7,22 @@ The [`cf-core`](../core) component is a dependency of this component."
   or your compiled Capital Framework CSS will
   not work perfectly in older browsers.
 
-## Vars
-
 ## Table of contents
 
 - [Variables](#variables)
-- [Striped table](#striped-table)
+- [Striped tables](#striped-tables)
 - [Right-aligned cells](#right-aligned-cells)
-- [Row Links](#row-links)
+- [Tables with row links](#tables-with-row-links)
 - [Sortable tables](#sortable-tables)
+  - [Making a table sortable](#making-a-table-sortable)
+  - [Sorting type](#sorting-type)
+  - [Sorting table on page load](#sorting-table-on-page-load)
+  - [Sample sortable table](#sample-sortable-table)
+  - [Important notes about sortable tables](#important-notes-about-sortable-tables)
 - [Responsive tables](#responsive-tables)
+  - [Responsive stacked table](#responsive-stacked-table)
+  - [Responsive stacked table with header](responsive-stacked-table-with-header)
+  - [Responsive table - horizontal scroll variation](#responsive-table---horizontal-scroll-variation)
 
 ## Variables
 
```

Comment:
Should we add a `### Color variables` section in here? I think one of the files above had a subsection, even when there was only one of the subsections.